modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -159,26 +159,26 @@ $ rm -f <installation_directory>/openshift/99_openshift-cluster-api_master-machi
 ----
 +
 By removing these files, you prevent the cluster from automatically generating control plane machines.
-endif::aws,azure,ash,gcp[]
+endif::azure,ash,gcp[]
 
-ifdef::aws,ash,azure,gcp[]
+ifdef::ash,azure,gcp[]
 . Remove the Kubernetes manifest files that define the control plane machine set:
 +
 [source,terminal]
 ----
 $ rm -f <installation_directory>/openshift/99_openshift-machine-api_master-control-plane-machine-set.yaml
 ----
-endif::aws,ash,azure,gcp[]
+endif::ash,azure,gcp[]
 
 ifdef::gcp[]
 ifndef::user-infra-vpc[]
 . Optional: If you do not want the cluster to provision compute machines, remove
 the Kubernetes manifest files that define the worker machines:
 endif::user-infra-vpc[]
 endif::gcp[]
-ifdef::aws,azure,ash,user-infra-vpc[]
+ifdef::azure,ash,user-infra-vpc[]
 . Remove the Kubernetes manifest files that define the worker machines:
-endif::aws,azure,ash,user-infra-vpc[]
```

Comment:
Azure looks correct, but AWS is missing some pieces.  We need to have all 3, as Azure does:

1. Remove the Kubernetes manifest files that define the control plane machines:

1. Remove the Kubernetes manifest files that define the control plane machine set:

1. Remove the Kubernetes manifest files that define the worker machines:


Right now, AWS only has (part of) Remove the Kubernetes manifest files that define the worker machines:

Reply:
Thanks for clarifying, Patrick. I added these steps back in. Additionally, I included an additional step after the `grep` command that now addresses the docs bug Jira update.